src/resources/js/components/CreatePresentation.vue

```diff
@@ -0,0 +1,128 @@
+<template>
+  <v-container>
+    <v-alert
+    v-model="message.show"
+    :class="message.type"
+    dismissible
+    >
+      {{ message.content }}
+    </v-alert>
+
+    <h2>Creation de presentation</h2>
+    <v-form
+      ref="form"
+      v-model="valid"
+      lazy-validation
+    >
+      <v-text-field
+        v-model="dataForm.title"
+        :counter="20"
+        :rules="titleRules"
+        label="Titre"
+        required
+      />
+
+      <v-menu
+        v-model="dateMenu"
+        :close-on-content-click="false"
+        :nudge-right="40"
+        transition="scale-transition"
+        offset-y
+        min-width="290px"
+      >
+        <template v-slot:activator="{ on }">
+          <v-text-field
+            v-model="dataForm.date"
+            label="Choisir une date"
+            :rules="dateRules"
+            required
+            readonly
+            v-on="on"
+          />
+        </template>
+        <v-date-picker v-model="dataForm.date" @input="menu = false" />
+      </v-menu>
+
+      <v-btn
+        :disabled="!valid"
+        color="success"
+        class="mr-4"
+        @click="createPresentation()"
+      >
+        Créer la présentation
+      </v-btn>
+
+    </v-form>
+  </v-container>
+</template>
+
+<script>
+import axios from 'axios';
+
+// set Bearer token in header of the future request
+axios.defaults.headers.common = { Authorization: `Bearer ${localStorage.getItem('Authorization-token')}` };
+
+export default {
+  data: () => ({
+    valid: true,
+    titleRules: [
+      (v) => !!v || 'Vous devez renseigner un titre',
+      (v) => (v && v.length <= 20) || 'Le Nom doit contenir moins de 20 caractères',
+    ],
+    dateMenu: false,
+    dateRules: [
+      (v) => !!v || 'Vous devez renseigner une date',
+    ],
+    dataForm: {
+      date: new Date().toISOString().substr(0, 10),
```

Comment:
the date field waits a date time see the api doc reference for more details

Reply:
ok I removed ``.toISOString().substr(0, 10)`` in [00ec8](https://github.com/kayoumido/HEIGVD-PRO-A-11/pull/129/commits/00ec8640675ed1a7b02dd02ee50de5e33c2d0592)